modal/_mount_utils.py

```diff
@@ -31,3 +32,25 @@ def validate_mount_points(
             raise InvalidError(f"{display_name} {path} cannot be mounted at '/tmp'.")
         validated.append((path, vol))
     return validated
+
+
+def validate_volumes(
+    volumes: Dict[Union[str, PurePosixPath], Union["_Volume", "_S3Mount"]]
+) -> List[Tuple[str, Union["_Volume", "_NetworkFileSystem", "_S3Mount"]]]:
+    if not isinstance(volumes, dict):
+        raise InvalidError("volumes must be a dict[str, Volume] where the keys are paths")
+
+    validated_volumes = validate_mount_points("Volume", volumes)
+    # We don't support mounting a volume in more than one location
+    volume_to_paths: Dict["_Volume", List[str]] = {}
+    for path, volume in validated_volumes:
+        if isinstance(volume, _Volume):
```

Comment:
This is correct, right? Or is it supposed to be `isinstance(volume, (_Volume, _NetworkFileSystem))`?